packages/react/stories/TextField.stories.tsx

```diff
@@ -26,10 +26,21 @@ Sizes.decorators = [
 
 export const Icons = Template.bind({});
 
+const onClick = () => {
+  console.log('Test');
```

Comment:
Please remove this console log. PR looks good otherwise

Reply:
+1

Reply:
We should just do a `return`